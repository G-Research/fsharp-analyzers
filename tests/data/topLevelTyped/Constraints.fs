module Sample

let areEqual (a: 'a) (b: 'b) : bool = a.Equals(b)

