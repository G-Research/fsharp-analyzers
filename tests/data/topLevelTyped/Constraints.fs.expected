GRA-UNTYPEDAPI-001 | Warning | (3,4 - 3,12) | `areEqual` is not private, not fully typed and not used outside `A.fs` in the current project.
Missing constraint: 'a : equality
Generic type parameter 'b not present.
